tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md

```diff
@@ -509,3 +507,15 @@ Other free text and markdown formatting can be used elsewhere in the document if
 
 ### [shares are not listed with full paths](https://github.com/owncloud/ocis/issues/2462)
 -   [webUISharingPublicBasic/publicLinkCreate.feature:88](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingPublicBasic/publicLinkCreate.feature#L88)
+
+### [web config update is not properly reflected after the ocis start](https://github.com/owncloud/ocis/issues/2944)
+-   [webUIFiles/breadcrumb.feature:50](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIFiles/breadcrumb.feature#L50)
+
+### Shares folder cannot be deleted and is always visible
+-  [webUIDeleteFilesFolders/deleteFilesFolders.feature:77](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIDeleteFilesFolders/deleteFilesFolders.feature#L77)
+-  [webUIDeleteFilesFolders/deleteFilesFolders.feature:89](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIDeleteFilesFolders/deleteFilesFolders.feature#L89)
+-  [webUISharingAcceptShares/acceptShares.feature:72](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingAcceptShares/acceptShares.feature#L72)
+-  [webUIOperationsWithFolderShares/accessToSharesFolder.feature:14](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIOperationsWithFolderShares/accessToSharesFolder.feature#L14)
+
+### [features/webUIPreview/imageMediaViewer.feature:57 is failing in CI and needs investigation](https://github.com/owncloud/web/issues/6275)
```

Comment:
@SwikritiT @phil-davis just want to once again voice my discomfort of just putting more stuff into expected failures instead of investigating the root cause

Reply:
These are already in https://github.com/owncloud/ocis/blob/master/tests/acceptance/expected-failures-webUI-on-OCIS-storage.md

They were added by: https://github.com/owncloud/ocis/commit/f542e58f773d946ab9a020e75c61b0a7efd09a09

Why was there not discomfort for that PR?

Reply:
There was, I did comment on it and asked for clarification whether it's wanted from a product perspective. But the tests referenced int his PR are totally different ones

Reply:
Would be good to keep media viewer green. Could you look into it and wait with the commit id bump if fixing the issue is not too complicated?

Reply:
I looked at the issue #6275 and posted more logs and thoughts. But it is not obvious why the test might be failing, and @SwikritiT reported that it passes locally for her. We will have to try harder tomorrow to see what the problem might be.

Reply:
here is how it can be reproduced https://github.com/owncloud/web/issues/6289
@SwikritiT is trying to find out what change started to cause it